data-access-layer/eventmesh/ApiServerClient.js

```diff
@@ -67,27 +67,26 @@ class ApiServerClient {
         return apiserver.loadSpec()
           .then(() => {
             this.ready = true;
-            logger.info('Loaded Successfully');
+            logger.info('Successfully loaded ApiServer Spec');
```

Comment:
Wanted to have an some sort of indicator that apiserver can take requests now.

Reply:
Right, but isnt it given that if there are no errors its ready? We have too many logs in PROD and all i try to do in my reviews is try to avoid logs which seem obvious and print only when needed. If you need the above for debug, please turn it debug level and also catch error and log at error level thats more easily seen in logs. 

Since we are undertaking refactoring code base its also a good time to ensure we get rid of unnecessary logs, which adds little value when troubleshooting.

Reply:
Done.